module/surveys/helper.js

```diff
@@ -1413,6 +1415,103 @@ module.exports = class SurveysHelper {
     });
   }
 
+  /**
+   * List of surveys based on UserId.
+   * @method
+   * @name listSurveyByUserId
+   * @param {String} requestUserId - id of the user to fetch docmunet.
+   * @param {String} queryType -     type of collection to query whether its survey or obs
+   * @returns {JSON} List of surveys.
+   */
+
+  static listSurveyByUserId(requestUserId, queryType = "") {
+    return new Promise(async (resolve, reject) => {
+      try {
+        let surveyDataofUser = await this.surveyDocuments(
+          { createdBy: requestUserId },
+          ["_id", "solutionId", "userId", "name", "createdBy"]
+        );
+
+        if (!(surveyDataofUser.length > 0)) {
```

Comment:
@praveenKDass if there is no data then it should then return big number as 0

Reply:
Removed the condition and returning 0 if there is no data